README.md

```diff
@@ -207,7 +207,7 @@ For this example, we add our new role to the playboodk as shown below:
       connection: chroot
       gather_facts: no
       vars:
-        ansible_python_interpreter: /usr/bin/python2
+        ansible_python_interpreter: /usr/bin/python3
```

Comment:
nit: is this still required? Wouldn't that be the default?

Reply:
Yes, that the default and I just reverted back to the default. All the playbooks have this line.